packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -166,6 +172,11 @@ export const resolvers = {
     ...pageLinkedEntities,
   },
 
+  Comment: {
+    properties: commentProperties,
+    ...commentLinkedEntities,
+  },
```

Comment:
If the `commentProperties` resolver is removed, there's no need for this field:
```suggestion
```

Reply:
Removed this in 9affaa36c8a93447844957915a8d38297236105c.